ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -2631,7 +2631,7 @@ static int zend_jit_setup(void)
 	/* To find offset of "_tsrm_ls_cache" in TLS segment we perform a linear scan of local TLS memory */
 	/* Probably, it might be better solution */
 	do {
-		void ***tls_mem = ((void***)__readfsdword(0x2c))[_tls_index];
+		void ***tls_mem = ((void**)__readfsdword(0x2c))[_tls_index];
```

Comment:
Shouldn't this be the other way around maybe? (Same above for X64)

```suggestion
		void **tls_mem = ((void***)__readfsdword(0x2c))[_tls_index];
```

Reply:
Oh, you're right! Thanks!